docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,3934 @@
+* Panel titles and descriptions are now transferred to the converted Lens panels in *TSVB* {kibana-pull}154713[#154713]
+* Adds the ability to use the empty label for `/` terms in *TSVB* {kibana-pull}154647[#154647]
+* Fixes the formatting for the legend actions title {kibana-pull}153747[#153747]
+* Adds support for negative filter ratios in *TSVB* {kibana-pull}152053[#152053]
+* Adds the ability to always retain source order for multi-metric partition chart layers in *Lens* {kibana-pull}151949[#151949]
+
+Machine Learning::
+* Data Frame Analytics/Anomaly Detection: Custom URLs - entity dropdown reflects Data View update {kibana-pull}155096[#155096]
+* AIOps: Fix race condition where stale url state would reset search bar {kibana-pull}154885[#154885]
+* Fixes anomalies table drilldown time range for longer bucket spans {kibana-pull}153678[#153678]
+* Do not match time series counter fields with aggs in wizards {kibana-pull}153021[#153021]
+* Anomaly Detection datafeed chart: ensure chart y axis minimum set correctly {kibana-pull}152051[#152051]
+
+Management::
+* Improves the display when there are many columns {kibana-pull}155119[#155119]
+* Fixes stale submit handler ref update {kibana-pull}154242[#154242]
+* Fixes terms aggregation support in wizard for Transforms {kibana-pull}151879[#151879]
+* Fixes an issue where you were unable to accept additional dynamic field values for an index template {kibana-pull}150543[#150543]
+
+Maps::
+* Fixes raster layer is missing in pdf/png exports {kibana-pull}154686[#154686]
+* Fixes RegionMap chart type does not work with reporting {kibana-pull}153492[#153492]
+* Fixes layers are not displayed in offline environment and map.includeElasticMapsService not set to false {kibana-pull}152396[#152396]
+
+Monitoring::
+Removes usage for the stats endpoint {kibana-pull}151082[#151082]
+
+Observability::
+* Adds space-specific feature privileges {kibana-pull}154734[#154734]
+* Adds the ability to properly handle NO DATA with multiple conditions with a mix of aggregations and document count thresholds {kibana-pull}154690[#154690]
+* Adds additional types to the fields to be use with cardinality aggregation for Metric Threshold Rule {kibana-pull}154197[#154197]
+* Adds persistent normalization mode {kibana-pull}153116[#153116]
+* Fixes refresh every in the alert search bar {kibana-pull}152246[#152246]
+
+Platform::
+Fixes badge counter for global settings {kibana-pull}150869[#150869]
+
+Querying & Filtering::
+* Adds the ability to unload a selected query when it is deleted {kibana-pull}154644[#154644]
+* Removes failures in wrong custom timerange {kibana-pull}154643[#154643]
+
+Reporting::
+* Fixes report generation when image panel is in the end of the layout {kibana-pull}153846[#153846]
+* Updates Chromium to 111.0.5555.0 (r1095492) and Puppeteer to 19.7.2 {kibana-pull}153033[#153033]
+
+Security::
+Fixes Session view and k8s dashboard {kibana-pull}154982[#154982]
```

Comment:
This probably belongs with the Elastic Security changelog as well

Reply:
@nastasha-solomon FYI

Reply:
Thanks, @KOTungseth! I'll note this as well.